src/Microsoft.Data.SqlClient/src/Microsoft/Data/OperationAbortedException.cs

```diff
@@ -18,7 +18,7 @@ private OperationAbortedException(string message, Exception innerException) : ba
             HResult = unchecked((int)0x80131936);
         }
 
-#if NET8_0_OR_GREATER
+#if NET
```

Comment:
There's no technical impact here, I'm just flagging up a difference in code style. When merging, @benrr101 and I have recently been using `#if !NETFRAMEWORK` for .NET Core-only code.

Reply:
Why even specify?